scripts/bootstrap.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Fetch dart dependencies
+dart pub global activate very_good_cli
+very_good --analytics=false
+very_good packages get -r ./packages
```

Comment:
using the recursive package install so we don't need to manually add new packages each time